react-frontend/public/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Senzr is a tool to track your indoor air quality"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon.png" />
+    <!--
```

Comment:
Do we need these comments here or can they be seen as redundant? I kind of want to clean up, but maybe the comments are helpful in the future. What do you think?

Reply:
Yeah! Go ahead and clean up comments as you wish. Create-react-app generates a lot of stuff that we can probably remove.